### ISSUE TITLE
Inconsistent output when using antd via parcel / webpack

### ISSUE THREAD

**Issue by babsonmatt** [parcel-bundler/parcel#109]
### 🎛 Configuration (.babelrc, package.json, cli command)

```js
{
  "presets": ["react-app"]
}

cli command: `NODE_ENV=development parcel index.html`

```

### 🤔 Expected Behavior - create-react-app
![image](https://user-images.githubusercontent.com/853220/33746993-856a4b44-db8e-11e7-8081-442e8ae00fbb.png)

### 😯 Current Behavior - parcel
![image](https://user-images.githubusercontent.com/853220/33747001-9293f5ea-db8e-11e7-9217-9016ede507d7.png)

### 🔦 Context
I'm not at all sure what's causing this issue, but for whatever reason I get the above inconsistencies in Chrome 63 (63.0.3239.84) when using create-react-app and parcel.  What's interesting is that Firefox produces the expected, correct output regardless of using create-react-app or parcel.

Working example (consistent with using create-react-app): https://stackblitz.com/edit/react-vl1ihs

### 🌍 Your Environment

| Software         | Version(s)
| ---------------- | ----------
| Parcel           |  1.0.3
| Node             | 9.2.0
| npm/Yarn         | yarn 1.3.2
| Operating System | osx 10.12

**Comment by davidnagli** [comment]
So are you trying to set up Create-React-App **and** Parcel, or are you trying to replace Create-React-App with Parcel?

**Comment by babsonmatt** [comment]
@davidnagli sorry for the lack of clarity there - working on setting up a new project with Parcel and just couldn't figure out what was causing the odd behavior so I recreated a simple test case with create-react-app and everything seems fine (also with codesandbox / stackblitz) so I'm stuck concluding that its something Parcel-ish?

Thanks for the reply!

**Comment by davidnagli** [comment]
Thanks for clarifying! I’m working on figuring out what’s wrong right now (I’ve been wrestling with stackblitz for the past 30 minutes)

**Comment by babsonmatt** [comment]
Thanks a lot @davidnagli, looking forward to using Parcel!  Let me know if I can help out or anything with the issue

**Comment by yuqianma** [comment]
Just to confirm my usage of antd is alright. But I notice I am using `antd 3.0.1`. Maybe it's an antd bug?

**Comment by davidnagli** [comment]
I wasn’t able to reproduce this locally outside of stackblitz

**Comment by yuqianma** [comment]
I mean everything runs correctly in my computer... including the stackblitz one. But the stackblitz uses antd 3.0.0. I guess it has been fixed in the latest 3.0.1.

**Comment by brandon93s** [comment]
Possibly related (released in 3.0.1): https://github.com/ant-design/ant-design/issues/8459

This _appears_ to be an `antd` issue and not a `parcel` issue. Please comment if this is still not fixed after updating to `antd@3.0.1`.
